packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -464,11 +464,27 @@ const waitForMediaLibrary = async ( driver ) => {
 	await waitForVisible( driver, locator );
 };
 
-const waitForVisible = async ( driver, elementLocator, iteration = 0 ) => {
-	const maxIteration = 25;
+/**
+ *
+ * @param {string} driver
+ * @param {string} elementLocator
+ * @param {boolean} returnBool - To return boolean instead of the first element/throw error. Useful for asserts.
```

Comment:
Yeah, I was conflicted about this one since I only needed the bool return value in one case, but I do agree to have it do multiple things can make it harder to understand. Thanks for your input! I will separate it and create a new `isElementVisible` that will return bool.

Reply:
This is done in this commit: https://github.com/WordPress/gutenberg/pull/40670/commits/46273a83845071272a8950c58365a30608581a1e